Update/omot_026.txt

```diff
@@ -793,13 +793,13 @@ void main()
 			NULL, " In the end, I couldn't even step foot on the battlefield.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 6, "ps3/s14/06/220600261", 256, TRUE);
 	OutputLine(NULL, "なんだか、性に合わないことしているなぁ」",
-			NULL, " I suppose that's more appropriate for a girl like me.\"", Line_Normal);
+			NULL, " I wonder if that's more appropriate for a girl like me.\"", Line_Normal);
 	ClearMessage();
 
 //r赤坂さんの頭に絆創膏を貼り付けながら、なんだか今の自分がおかしくて愚痴が出る。本当はお姉や圭ちゃんたちみたく、鉄火場に乗り込んで大暴れするほうが私らしくて気分がいいのだけど。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "赤坂さんの頭に絆創膏を貼り付けながら、なんだか今の自分がおかしくて愚痴が出る。本当はお姉や圭ちゃんたちみたく、鉄火場に乗り込んで大暴れするほうが私らしくて気分がいいのだけど。",
-			NULL, "Somehow, I found myself making weird complaints as I pasted a bandage to Akasaka-san's head. I really did want to see Sis and Kei-chan on the battlefield, and I'd feel better going on a rampage over there with them.", Line_Normal);
+			NULL, "Somehow, I found myself making weird complaints as I pasted a bandage to Akasaka-san's head. Actually, I'd feel better going on a rampage on the battlefield like Sis and Kei-chan and it suits me though.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Somehow, I found myself making weird complaints as I pasted a bandage to Akasaka-san's head. Really, I'd feel better going on a rampage on the battlefield like Sis and Kei-chan, and it would suit me.", Line_Normal);
```